### PR TITLE
Fix/react 18 freezes

### LINKED ISSUE #235: Freeze app with using React 18
**Describe the bug**
After click selector/dropdown/datepicker the app is freezing (potential antd issue with react 18). 

**To Reproduce**
1. Go to medical dashboard
2. Click on selector to choose new month

**Expected behavior**
After click popup is open with the list of months.

**Screenshots**
Screenshot from 2022-08-19 17-56-42

**Desktop (please complete the following information):**
 - OS: ubuntu 22.04
 - Browser: chrome

**Additional context**
With React 18 this happens intermittently on some pages

### PR DESCRIPTION
**Pull request by saha80**
# Description

Fixes #235 

## Type of change

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Locally.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/common/BaseHashTag/BaseHashTag.styles.ts [hunk 1]
```diff
@@ -16,7 +15,15 @@ export const RemoveTagIcon = styled(CloseOutlined)`
   cursor: pointer;
 `;
 
-export const TagWrapper = styled(TagSpan)`
+// https://github.com/rcdexta/react-trello/blob/68cfec88a14312da00a5ca0e1a29bc8de40321d8/src/styles/Base.js#L210
+export const TagSpan = styled.span`
```
Comment (leineveber) [inline src/components/common/BaseHashTag/BaseHashTag.styles.ts:19]: This is unused component, can be removed

File: tsconfig.json [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx"
-  },
-  "include": [
-    "src"
-  ]
-}
+{
```
Comment (leineveber) [inline tsconfig.json:1]: what's changed in this file?

**Review by leineveber — APPROVED** [review]

File: tsconfig.json (hunk 2 above)
Comment (saha80) [reply tsconfig.json:1]: End of line. react-scripts@4.0.3 rewrites tsconfig each run and changes end of line to the operating system EOL.

https://github.com/facebook/create-react-app/blob/f92c37a6e6636ed1650ffd976c6881f59b1be803/packages/react-scripts/scripts/start.js#L31

https://github.com/facebook/create-react-app/blob/f92c37a6e6636ed1650ffd976c6881f59b1be803/packages/react-scripts/scripts/utils/verifyTypeScriptSetup.js#L37

File: src/components/common/BaseHashTag/BaseHashTag.styles.ts (hunk 1 above)
Comment (saha80) [reply src/components/common/BaseHashTag/BaseHashTag.styles.ts]: Removed

**Pull request merged**